laravel/Caddyfile

```diff
@@ -0,0 +1,17 @@
+example.com {
+
+    root ./public
+    log ./storage/logs/caddy-access.log
+    errors ./storage/logs/caddy-error.log
+
+    fastcgi / 127.0.0.1:9000 php {
+        index index.php
+    }
+    
+    rewrite {
+        r .*
```

Comment:
Update: Abiola suggested a more concise rewrite in the forum; maybe use that, if it works!